microprofile/lra/client/spi/src/main/java/module-info.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+module io.helidon.microprofile.lra.coordinator.client {
+    requires transitive java.xml.bind;
```

Comment:
Why is XML binding required by this module?

Reply:
Its required by `java.ws.rs` module 2.1.6 [module-info.java](https://github.com/eclipse-ee4j/jaxrs-api/blob/2.1-Maintenance/jaxrs-api/src/main/java/module-info.java) 

But it shoud get optional since 3.* https://github.com/eclipse-ee4j/jaxrs-api/issues/907